webview/util.py

```diff
@@ -100,7 +100,7 @@ def base_uri(relative_path: str = '') -> str:
 
 
 def create_cookie(input_: dict[Any, Any] | str) -> SimpleCookie[str]:
-    if isinstance(type, dict):
+    if isinstance(input_, dict):
```

Comment:
What tests are failing?

Reply:
Those test fails are semi-random and have been occurring for a while. Do not worry about them.
Would you resolve conflicts? Otherwise the PR is good to go

Reply:
Yes please